change/@fluentui-react-infobutton-e5984e77-89ae-4c61-85e0-984121adb4e5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27316&commitId=7f8708fb5c8b4d0af2c619481a6385807b48009a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1899 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - Dark Mode.active.chromium.png|1108| Changed|
|InfoButton.default - High Contrast.active.chromium.png|1263| Changed|
|InfoButton.default.active.chromium.png|1771| Changed|
|InfoButton.size.rest.chromium.png|3374| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>